cpp/open3d/t/geometry/PointCloud.cpp

```diff
@@ -1369,6 +1369,16 @@ core::Tensor PointCloud::ComputeMetrics(const PointCloud &pcd2,
                                 metrics, params);
 }
 
+bool PointCloud::IsGaussian() const {
+    std::vector<std::string> keys_to_check = {"f_dc", "f_rest", "normals", "opacity", "rot", "scale"};
```

Comment:
Can you rename to `IsGaussianSplat`?
We should skip "f_rest" from this list. f_rest is optional.